src/classes/settings.py

```diff
@@ -39,6 +41,18 @@ class SettingStore(JsonDataStore):
     """ This class only allows setting pre-existing keys taken from default settings file, and merges user settings
     on load, assumes default OS dir."""
 
+    @unique
+    class pathType(Enum):
+        RECENT=0 # Return the last used path for the same action
```

Comment:
Is there a reason we are starting with 1 here, but using auto() in other places?

Reply:
My reason was just to ensure that the value of RECENT and PROJECT were the same as the value of their respective settings.

Reply:
But we could change the settings to use 1, and 2 as values, right?

Reply:
Yeah, No reason the setting values can't be 1 and 2